drivers/sensor/bme280/bme280.c

```diff
@@ -173,6 +173,53 @@ static int bme280_sample_fetch(struct device *dev, enum sensor_channel chan)
 	if (data->chip_id == BME280_CHIP_ID) {
 		size = 8;
 	}
+
+#if defined CONFIG_BME280_FORCED_MODE_ON
```

Comment:
Can shorten it a bit with `#ifdef CONFIG_BME280_FORCED_MODE`.

Reply:
Done